tests/io_perfomance/image-volume/config.fio

```diff
@@ -0,0 +1,182 @@
+[global]
+ioengine=libaio
+size=1G
+direct=1
+runtime=60
+time_based=1
+group_reporting
+filename=/data/fio.test.file
+
+#SEQ READ 4k
+[read-4k-1]
+rw=read
+bs=4k
+iodepth=16
+numjobs=1
+stonewall
+
+[read-4k-2]
+rw=read
+bs=4k
+iodepth=16
+numjobs=8
+stonewall
+
+[read-4k-3]
+rw=read
+bs=4k
+iodepth=32
+numjobs=8
+stonewall
+
+[read-4k-4]
+rw=read
+bs=4k
+iodepth=32
+numjobs=1
+stonewall
+
+#SEQ READ 64k
+[read-64k-1]
+rw=read
+bs=64k
+iodepth=16
+numjobs=1
+stonewall
+
+[read-64k-2]
+rw=read
+bs=64k
+iodepth=16
+numjobs=8
+stonewall
+
+[read-64k-3]
+rw=read
+bs=64k
+iodepth=32
+numjobs=8
+stonewall
+
+[read-64k-4]
+rw=read
+bs=64k
+iodepth=32
+numjobs=1
+stonewall
+
+#SEQ READ 1m
+[read-1m-1]
+rw=read
+bs=1m
+iodepth=16
+numjobs=1
+stonewall
+
+[read-1m-2]
+rw=read
+bs=1m
+iodepth=16
+numjobs=8
+stonewall
+
+[read-1m-3]
+rw=read
+bs=1m
+iodepth=32
+numjobs=8
+stonewall
+
+[read-1m-4]
+rw=read
+bs=1m
+iodepth=32
+numjobs=1
+stonewall
+
+#SEQ WRITE 4k
+[write-4k-1]
+rw=write
+bs=4k
+iodepth=16
+numjobs=1
+stonewall
+
+[write-4k-2]
+rw=write
+bs=4k
+iodepth=16
+numjobs=8
+stonewall
+
+[write-4k-3]
+rw=write
+bs=4k
+iodepth=32
+numjobs=8
+stonewall
+
+[write-4k-4]
+rw=write
+bs=4k
+iodepth=32
+numjobs=1
+stonewall
+
+#SEQ WRITE 64k
+[write-64k-1]
+rw=write
+bs=64k
+iodepth=16
+numjobs=1
+stonewall
+
+[write-64k-2]
+rw=write
+bs=64k
+iodepth=16
+numjobs=8
+stonewall
+
+[write-64k-3]
+rw=write
+bs=64k
+iodepth=32
+numjobs=8
+stonewall
+
+[write-64k-4]
+rw=write
+bs=64k
+iodepth=32
+numjobs=1
+stonewall
+
+#SEQ WRITE 1m
+[write-1m-1]
+rw=write
+bs=1m
+iodepth=16
+numjobs=1
+stonewall
+
+[write-1m-2]
+rw=write
+bs=1m
+iodepth=16
+numjobs=8
+stonewall
+
+[write-1m-3]
+rw=write
+bs=1m
+iodepth=32
+numjobs=8
+stonewall
+
+[write-1m-4]
+rw=write
+bs=1m
+iodepth=32
+numjobs=1
+stonewall
```

Comment:
While a lot of these settings look reasonable, I wonder where did they come from. Lets discuss how did you decide on picking these very settings and not something else.